jose/cryptphy.py

```diff
@@ -0,0 +1,112 @@
+import six
+
+from cryptography.hazmat.backends import default_backend
```

Comment:
Are all the `hazmat` imports necessary because cryptography doesn't (yet?) expose these algorithms in their 'public' layer?